common/persistence/sql/sqlplugin/mysql/execution.go

```diff
@@ -142,6 +142,17 @@ ORDER BY task_id LIMIT ?`
 	deleteVisibilityTaskQuery      = `DELETE FROM visibility_tasks WHERE shard_id = ? AND task_id = ?`
 	rangeDeleteVisibilityTaskQuery = `DELETE FROM visibility_tasks WHERE shard_id = ? AND task_id > ? AND task_id <= ?`
 
+	createTieredStorageTasksQuery = `INSERT INTO tiered_storage_tasks(shard_id, task_id, data, data_encoding) 
+ VALUES(:shard_id, :task_id, :data, :data_encoding)`
+
+	getTieredStorageTaskQuery = `SELECT task_id, data, data_encoding 
+ FROM tiered_storage_tasks WHERE shard_id = ? AND task_id = ?`
+	getTieredStorageTasksQuery = `SELECT task_id, data, data_encoding 
+ FROM tiered_storage_tasks WHERE shard_id = ? AND task_id > ? AND task_id <= ? ORDER BY shard_id, task_id`
```

Comment:
`ORDER BY shard_id, task_id` -> `ORDER BY task_id`

Reply:
plz also make similar change to line 93 & 140

Reply:
Done